doc/source/how-to/dns-configuration.rst

```diff
@@ -0,0 +1,91 @@
+DNS configuration
+=================
+
+As explained in :ref:`Documenting`, PyAnsys projects publish their documentation
+online under the following canonical name (CNAME) convention:
+
+``https://<product>.docs.pyansys.com``
+
+To request a CNAME for the ``pyansys.com`` domain, please contact
+`Maxime Rey`_, `Roberto Pastor Muela`_ or `Alex Kaszynski`_. Any of these members
```

Comment:
Given that this guide is public-facing, do we want to use specific names or use the PyAnsys support email address?

Reply:
This specific part of the documentation, even if it's public facing, is directed for Ansys employees only, since they are the ones that can request CNAME creations. I think it's fine if we leave it as is.